packages/common.vm/common.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>common.vm</id>
-    <version>0.0.0.20220113</version>
+    <version>3.0.1.20220222</version>
```

Comment:
lol can't remember, but why did we change it from a metapackage version to a non-metapackage version? Was this to make merge easier and we'd change later?

Reply:
typo, fixed!